src/widgets/Window.cpp

```diff
@@ -722,6 +737,27 @@ void Window::addMenuBar()
     connect(prevTab, &QAction::triggered, this, [this] {
         this->notebook_->selectPreviousTab();
     });
+
+    // Help menu.
+    QMenu *helpMenu = mainMenu->addMenu(QString("Help"));
+
+    // Help->Chatterino Wiki item
+    QAction *helpWiki = helpMenu->addAction(QString("Chatterino Wiki"));
+    connect(helpWiki, &QAction::triggered, this, []() {
+        QDesktopServices::openUrl(QUrl("https://wiki.chatterino.com"));
+    });
+
+    // Help->Chatterino Github
+    QAction *helpGithub = helpMenu->addAction(QString("Chatterino Github"));
+    connect(helpGithub, &QAction::triggered, this, []() {
+        QDesktopServices::openUrl(QUrl("https://github.com/Chatterino/chatterino2"));
+    });
+
+    // Help->Chatterino Discord
+    QAction *helpDiscord = helpMenu->addAction(QString("Chatterino Discord"));
+    connect(helpDiscord, &QAction::triggered, this, []() {
+        QDesktopServices::openUrl(QUrl("https://discord.gg/7Y5AYhAK4z"));
```

Comment:
This should use the same constant as the link that's currently in the AboutPage (`LINK_CHATTERINO_DISCORD`). Should this expire or need to be deleted, changing it shouldn't be unnecessarily hard.

Reply:
Will do!

Reply:
@Mm2PL what would, in your opinion, be the best option to have that same constant (`LINK_CHATTERINO_DISCORD`) from `AboutPage.cpp` in `Window.cpp` as well?

I am still wrapping my head around the code base and haven't written C++ in a long time so I wanted to have some suggestions before I make any changes that may go beyond just the menu bar items.

Here are a couple of options:
* Create a header file in `src/common/ProjectInfo.hpp` which holds those similar constants that other things can include? (likely option and probably good for future.)
* Include `AboutPage.cpp` and use that in `Window.cpp` (probably the least preferred way?)
* Create the same constant in `Window.cpp` (duplication)

Reply:
You could move the `LINK_CHATTERINO_DISCORD` definition to `src/common/Common.hpp` & use that in both files